backend/src/config.yaml

```diff
@@ -0,0 +1,10 @@
+# config.yaml
+
+questionsFile: "test-questions.json"                                                  # File containing questions in JSON format
+couponsFile: "test-coupons.txt"                                                       # File containing coupons, one per line
+metadataFile: "" #"../frontend/src/assets/images/metadata_ipfs.json"                      # Metadata File
```

Comment:
```suggestion
nftMetadataFile: "" #"../frontend/src/assets/images/metadata_ipfs.json"                      # Metadata File
```

Reply:
Added under nftReward in test-config.yaml.